content/pt-br/docs/concepts/configuration/windows-resource-management.md

```diff
@@ -0,0 +1,75 @@
+---
+reviewers:
+- jayunit100
+- jsturtevant
+- marosset
+- perithompson
+title: Gerenciamento de recursos para nós do Windows
+content_type: concept
+weight: 75
+---
+
+<!-- overview -->
+
+Esta página descreve as diferenças em como os recursos são gerenciados entre o Linux e o Windows.
+
+<!-- body -->
+
+Em nós do Linux, {{< glossary_tooltip text="cgroups" term_id="cgroup" >}} são usados como um limite de pod para controle de recursos. 
+Os contêineres são criados dentro desse limite para o isolamento de rede, processo e sistema de arquivos. 
+As APIs Linux cgroup podem ser usadas para coletar estatísticas de uso de CPU, E/S e memória.
+
+Em contraste, o Windows usa um [_objetos de trabalho_](https://docs.microsoft.com/windows/win32/procthread/job-objects) por contêiner com um filtro de namespace do sistema
+para conter todos os processos em um contêiner e fornecer isolamento lógico ao hospedar.
+(Os objetos de trabalho são um mecanismo de isolamento de processo do Windows e são diferentes dos
+que o Kubernetes chama de {{< glossary_tooltip term_id="job" text="Job" >}}).
+
+Não há como executar um contêiner do Windows sem a filtragem de namespace. 
+Isso significa que os privilégios do sistema não podem ser declarados no contexto do host e, 
+portanto, os contêineres privilegiados não estão disponíveis no Windows.
+Os contêineres não podem assumir uma identidade do host porque o Gerente de conta de segurança (SAM) é separado.
+
+## Gerenciamento de memória {#resource-management-memory}
+
+O Windows não possui um eliminador de processo de falta de memória como o Linux. 
+O Windows sempre trata todas as alocações de memória do modo de usuário como 
+virtuais e os arquivos de paginação são obrigatórios.
+
+Os nós do Windows não sobrecarregam a memória para os processos. O efeito líquido 
+é que o Windows não atingirá as condições de falta de memória 
+da mesma forma que o Linux, e processará a página em disco em vez de estar 
+sujeito ao encerramento por falta de memória (OOM). Se a memória for 
+superprovisionada e toda a memória física estiver esgotada, a paginação poderá diminuir o desempenho.
```

Comment:
Some bits here:
* We have used "superdimensionamento" for overcommitment before (see https://kubernetes.io/pt-br/docs/concepts/configuration/manage-resources-containers/#tipos-de-recursos). How do you feel about using it here too (i.e. "...nós Windows não superdimensionam...")?
* I see why you used "efeito líquido" in this context, but I think it sounds a bit strange (it looks more commonly used in the context of economics). Perhaps "efeito real", or even just "efeito" would sound better?
* In the "processes page to disk" sentence, it appears to me that "page" is the verb, not "processes" (context of memory paging). I unfortunately don't have suggestions to give on this one, but the translated sentence sounds a bit out-of-context to me.

Reply:
My suggestion was to put this sentence: "and will be processing the page on disk". Does it feel right?